### DISCUSSION TITLE
[Bug Report] Error generating Phash

### DISCUSSION THREAD

**Discussion by Aiki033** [stashapp/stash#5548] [Q&A] [answered]
**Describe the bug**
In system duplicator it says "Missing Phashes for 48 scenes". so I tried generating phash but I get this error. it says generate finished but when I check tools - - > scene duplicator checker, it still says 48 phashes are still missing.
Error : command stderr: [hevc @ 0xffffa2b67870] VPS 0 does not exist
[hevc @ 0xffffa2b67870] SPS 0 does not exist.
[hevc @ 0xffffa0b81820] VPS 0 does not exist
[hevc @ 0xffffa0b81820] SPS 0 does not exist.
[hevc @ 0xffffa0b81820] PPS id out of range: 0
[vist#0:0/hevc @ 0xffffa589ab10] Error submitting packet to decoder: Invalid data found when processing input

**To Reproduce**
1. Go to 'Task'
2. Click on 'Generate (I tried leaving only phash enabled)'

**Expected behavior**
Generate phashes.

**Screenshots**
![image](https://github.com/user-attachments/assets/1ed3df69-67e4-487a-800a-8c291dbf6d1e)

**Stash Version: v0.27.2

**Server: SBC (Orangepi5) stashapp running on Docker
 - OS: Armbian 12
 - Browser
 - Version
 
**Additional context**
I don't know what happened. I just added my few new videos and tried to generate phashes and I started to get this error.

**Comment by stg-annon** [comment, marked as answer]
Some of your video files are probably slightly corrupt: most media players will play them just fine (and skip/compensate for the broken parts) but Stash needs them to be flawless to generate a phash

**Comment by Aiki033** [comment]
a week ago, I used tdarr to convert some large files to HEVC using the tdarr's default plugin stack without any errors. I guess somewhere in the process, it corrupted these video files.

**Reply by stg-annon** [reply]
could be that there was an issue with the reencode process with a few files, phash generation is kind of particular and will fail on almost minor issues with a file that are recoverable with most playback clients
